server/config/default.json

```diff
@@ -1,7 +1,10 @@
 {
   "global": {
     "loglevel": "debug",
-    "sessionSecret": "1DVETLX8VE007RYQN9TN9Z00"
+    "sessionSecret": "1DVETLX8VE007RYQN9TN9Z00",
+    "FrontPopupEnable": false,
+    "FrontPopupTitle": "",
+    "FrontPopupContent": ""
```

Comment:
minor annoying thing - can we change those vars to `frontPopup...` for consistency :D

Reply:
Fixed